openfe/setup/lomap_mapper.py

```diff
@@ -55,3 +56,52 @@ def _mappings_generator(self, mol1, mol2):
 
         yield AtomMapping(mol1=mol1, mol2=mol2, mol1_to_mol2=mapping_dict)
         return
+
+    def _get_mcs(self, mapping):
+        # get mcs from cache, else create and place into cache
+        try:
+            mcs = self._mcs_cache[mapping]
+        except KeyError:
+            mcs = lomap_mcs.MCS(mapping.mol1.rdkit, mapping.mol2.rdkit,
+                                self.time, threed=self.threed,
+                                max3d=self.max3d)
+            self._mcs_cache[mapping] = mcs
+        return mcs
```

Comment:
The object we need from MCS is the `mcs_mol`.  There might be a shortcut from using the indices in the mapping to recreate this without rerunning the MCS alg.